src/qunit.cpp

```diff
@@ -1068,6 +1123,10 @@ void QUnit::CNOT(bitLenInt control, bitLenInt target)
 
     QEngineShard& cShard = shards[control];
     QEngineShard& tShard = shards[target];
+
+    RevertBasis2(control);
```

Comment:
Naming things with a '1' or '2' at the end often confuses subsequent readers and developers - can a more expressive name be identified?

At the least, commentary needs to be provided at definition as to what's different about this variant of the function.

Reply:
Changed to "...1Qb" and "...2Qb."